src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx

```diff
@@ -1,42 +1,62 @@
 import React from 'react';
-import { get } from 'lodash-es';
+import { get, map } from 'lodash-es';
 
 import TreeEditor from './TreeEditor';
 import ConfigureDefaultField from './ConfigureDefaultField';
+import RelationshipEditor from './RelationshipEditor';
 
-export function RelationshipEditor() {
-  return null;
-}
+export function RelationshipEditorWrapper({
+  onClose,
+  onSubmit,
+  formSettings,
+  setFormSettings,
+  children,
+  ...rest
+}) {
+  const relationshipOptions = formSettings
+    ? get(formSettings, 'relationships')
+    : [];
 
-// export function RelationshipEditor({
-//   onClose,
-//   onSubmit,
-//   formSettings,
-//   setFormSettings,
-//   ...rest
-// }) {
-//   const speciesOptions = get(formSettings, 'relationships', []);
+  const transformedRelationshipOptions = map(
+    relationshipOptions,
+    (relationshipArr, key) => {
+      const currentObj = {};
+      currentObj.category = key;
+      currentObj.relationships = relationshipArr;
+      return currentObj;
+    },
+  );
 
-//   return (
-//     <OptionEditor
-//       open
-//       onClose={onClose}
-//       onSubmit={onSubmit}
-//       schema={{
-//         labelId: 'EDIT_RELATIONSHIPS',
-//         descriptionId: 'EDIT_RELATIONSHIPS_DESCRIPTION',
-//       }}
-//       value={speciesOptions}
-//       onChange={newOptions => {
-//         setFormSettings({
-//           ...formSettings,
-//           relationships: newOptions,
-//         });
-//       }}
-//       {...rest}
-//     />
-//   );
-// }
+  return (
+    <ConfigureDefaultField
+      onClose={onClose}
+      onSubmit={() => {
+        onSubmit();
+      }}
+      open
+    >
+      <RelationshipEditor
+        schema={{ labelId: 'RELATIONSHIPS' }}
+        value={transformedRelationshipOptions}
+        onChange={relationships => {
+          const formSettingObj = {};
+          const reformattedRelationships = relationships.reduce(
+            (memo, obj) => {
+              let currentObj = {};
+              currentObj[(obj?.category)] = obj?.relationships;
```

Comment:
Just undoing the thing from above on the way back out, as it were.

Reply:
I'd really rather not do this if it's not necessary :D

Reply:
Done